lib/teams.rb

```diff
@@ -0,0 +1,7 @@
+class Team
+
+  def initialize(params)
+    @params = params
```

Comment:
@params instance var exists only in this commit. Replaced with individual variables in later commits.